### DISCUSSION TITLE
Update a stored threejs's Vector

### DISCUSSION THREAD

**Discussion by TheoHiwo** [pmndrs/zustand#1830] [Q&A]
How to tell zustand that a variable myVector of type Vector3 needs to render components when setMyVector( state.myVector.set(x, y, z) ) is called?
For now, myVector is an array [x, y, z], but I'd like to know if there is another way or if this is good practice to store a Vector. 
Thanks for your consideration.

**Comment by dai-shi** [comment]
I think you need to clone it.
```js
setMyVector(state.myVector.clone().set(...))
```
